# This defines a waypoint along a trajectory.
float64 time                                # Point in time at which to reach the state
float64[] position                          # Desired position (rad or m) for given time
float64[] velocity                          # Desired speed (rad/s or m/s) for given time
float64[] acceleration                      # Desired acceleration (rad/s/s or ms/s/s) for given time
